src/page-modules/departures/details/details.module.css

```diff
@@ -0,0 +1,100 @@
+.container {
+  margin: 0 auto;
+  max-width: var(--maxPageWidth);
+  padding: var(--spacings-xLarge);
+
+  display: grid;
+  grid-template-columns: 2fr 3fr;
+  grid-template-areas:
+    'header header'
+    'serviceJourney map';
+  gap: var(--spacings-small);
+}
+@media (max-width: 650px) {
+  .container {
+    grid-template-columns: 1fr;
+    grid-template-areas:
+      'header'
+      'serviceJourney'
+      'map';
+  }
+}
+.headerContainer {
+  grid-area: header;
+  display: flex;
+  flex-direction: column;
+  gap: var(--spacings-large);
+}
+.mapContainer {
+  grid-area: map;
+  max-height: 37.5rem;
+  width: 100%;
+  margin: 0 auto;
+}
+.serviceJourneyContainer {
+  grid-area: serviceJourney;
+}
+@media (max-width: 650px) {
+  .serviceJourneyContainer {
+    margin: 0 auto;
+  }
+}
+
+.header {
+  display: flex;
+  align-items: center;
+  gap: var(--spacings-large);
+}
+
+.realtimeText {
+  display: flex;
+  align-items: center;
+  gap: var(--spacings-small);
+  font-size: 0.875rem;
+  color: var(--text-colors-secondary);
+}
+
+.callRows {
+  margin-top: var(--spacings-large);
+}
+.callRows__container {
+  padding: var(--spacings-medium);
```

Comment:
In the design the spacing between the first group(?) and the second is `1.25rem`, this padding adds up to `1.5rem`. Not too big of a deal I think.

<img width="148" alt="image" src="https://github.com/AtB-AS/planner-web/assets/14045515/aa1b412c-2ac2-4760-abac-e4579db71a30">

Reply:
Would have to change the padding to `0.625rem` instead of `var(--spacings-medium)` then 🤔

Reply:
Or could do some css fanciness. `padding-bottom: var(spacings-large);` on first-child, and `padding-top: 0;` on last-child. Not sure if that's a better solution than accepting a larger gap though. And maybe there are more potential callRows__containers inbetween?

Reply:
This has been solved now, thanks for the help on Slack @adriansberg 🎉